torch/_dynamo/eval_frame.py

```diff
@@ -563,6 +565,56 @@ def call_compiled_autograd():
 
             self.enter_exit_hooks.append(call_compiled_autograd)
 
+    def __call__(self, fn):
```

Comment:
Should we unconditionally make this change, or only do it when config.warmup_runs > 0?

Reply:
This seems like the wrong place to put it, why is this not in _TorchDynamoContext, or better yet, in the compiler itself?